### DISCUSSION TITLE
Change automatic Tag

### DISCUSSION THREAD

**Discussion by lordratner** [tteck/Proxmox#2242] [Ideas]
This is just a nit-pick, but I figured I'd suggest it.

The tag generated is "proxmox-helper-scripts" is soooo long. I'm changing mine to "tteck" but anything short would do. Not sure if others agree, so I figured I'd post here and see what others think.

**Comment by tteck** [comment]
It's all about the color,  😉  "proxmox-helper-scripts" is blue while "tteck" is pink.

**Reply by lordratner** [reply]
You want pink, you got pink:
![tteck](https://github.com/tteck/Proxmox/assets/49489398/b64dcb6f-622e-43fe-9610-f5cb729db35f)

But "proxmox-helper-scripts" is what all mine were auto tagged with, and that was just way too many letters 🤣
